packages/shared/src/components/CompetitiveStats/CompStats.tsx

```diff
@@ -220,7 +225,9 @@ export default function CompStats(props: { activeBracket: string; sortKey: strin
                       </div>
                     </th>
                     <td className="bg-base-200 text-right">{stats.total}</td>
-                    <td className="bg-base-200 text-right">{(stats.winRate * 100).toFixed(1)}%</td>
+                    <td className="bg-base-200 text-right">
+                      {(stats.winRate * winRateCorrectionFactor * 100).toFixed(1)}%
```

Comment:
Is the correction factor better off as a product or an additive factor? I can't really think of a good reason to go with one over the other but curious if you did